geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/executor/sortedset/SortedSetLexRangeOptions.java

```diff
@@ -83,13 +83,19 @@ void handleWithScoresArgument() {
   @Override
   int compareStartToEnd() {
     if (isRev) {
-      // TODO: implement ZREVRANGEBYLEX
-    }
-    int dummyNameComparison = checkDummyMemberNames(start.value, end.value);
-    if (dummyNameComparison == 0) {
-      return javaImplementationOfAnsiCMemCmp(start.value, end.value);
+      int dummyNameComparison = checkDummyMemberNames(end.value, start.value);
```

Comment:
consider extracting this block of code into a method that take two values. Then all you need to do differently (based on isRev) is pass it end+start vs. start+end.